bin/qlf_pipeline.py

```diff
@@ -267,17 +263,13 @@ def was_processed(self):
         'night': '20170428',
         'expid': '3',
         'zfill': '00000003',
-        'data_dir': '/home/singulani/raw_data',
+        'desi_spectro_data': '/home/singulani/raw_data',
```

Comment:
Please use argparse to create options to pass in from the command line instead of hardcoding specific developers directories into the file.